src/main/java/com/codehusky/huskyui/states/action/Action.java

```diff
@@ -31,10 +32,14 @@
  */
 public class Action {
 
-    @Nonnull private final StateContainer container;
-    @Nonnull private final Player observer;
-    @Nonnull private final ActionType type;
-    @Nonnull private final String goalState;
+    @Nonnull
```

Comment:
While I appreciate people having differing opinions on formatting, this change makes the rest of the classes I rewrote inconsistent with this single file.